pytential/symbolic/mappers.py

```diff
@@ -137,6 +139,66 @@ def map_interpolation(self, expr):
 
         return type(expr)(expr.from_dd, expr.to_dd, operand)
 
+# }}}
+
+
+# {{{ IntGTermCollector
+
+class IntGTermCollector(IdentityMapper):
```

Comment:
Having these mappers as part of the global `mappers` module is confusing to my mind. Maybe move them someplace more direct-solver-specific? If a more generic use case shows up, they can always be fished out.

Reply:
Moved to `pytential.linalg.direct_solver_symbolic` in latest version